tests/nnstreamer_sink/unittest_sink.cc

```diff
@@ -411,6 +413,25 @@ _test_src_eos_timer_cb (gpointer user_data)
   return FALSE;
 }
 
+/**
+ * @brief Wait until the pipeline processing the buffers
+ * @return TRUE on success, FALSE when a time-out occurs
+ */
+static gboolean
+_wait_pipeline_process_buffers (guint expected_num_buffers)
+{
+  gint timer_count = 0;
+  /* Waiting for expected buffers to arrive */
+  while (g_test_data.received < expected_num_buffers) {
+    timer_count++;
+    g_usleep (10000);
```

Comment:
Where '10000' come from?

Reply:
There is no special reason. I set it because I thought it was appropriate with one interval.

Reply:
Then, how about use a const variable for better readability?

Reply:
Thank you for suggestion. Updated!